libs/ui/src/modal.tsx

```diff
@@ -193,8 +195,11 @@ export function Modal({
       overlayClassName="_"
     >
       <ModalContent centerContent={centerContent} fullscreen={fullscreen}>
-        {title && <H2 as="h1">{title}</H2>}
-        {content}
+        {disableAutoplayAudio ? (
+          modalContent
+        ) : (
+          <ReadOnOpen>{modalContent}</ReadOnOpen>
```

Comment:
Double checking that using `ReadOnOpen` on non-voter facing machines, like VxAdmin, is safe. I imagine it'll just be a no-op on those machines

Reply:
Yeah, it'll be a no-op in cases where there's no `UiStringsContextProvider` at the root, but I should probably make that a little more obvious in the code -- will add a conditional or comment to clarify